roles/cockpit-ui/tasks/main.yml

```diff
@@ -1,86 +1,37 @@
 ---
-- name: Create temp directory for kubeconfig
-  command: mktemp -d /tmp/openshift-ansible-XXXXXX
-  register: mktemp
-  changed_when: False
-
-- set_fact:
-    openshift_hosted_kubeconfig: "{{ mktemp.stdout }}/admin.kubeconfig"
```

Comment:
This is where the actual issue is I believe. ``openshift_hosted`` didn't report an error AFAICT but ``cockpit-ui`` did due to the use of command.